src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -1794,7 +1765,7 @@ private boolean isAccessAuthorized(DataFile df, String apiToken) {
                 return false;
             } 
             
-        if (permissionService.requestOn(createDataverseRequest(user), df).has(Permission.DownloadFile)) { 
+        if (permissionService.requestOn(createDataverseRequest(user), df).has(Permission.DownloadFile)) { // FIXME: Indent this line more.
```

Comment:
Can this just be changed? (was this a test to see if reviewers would notice :-) ? )

Reply:
Fixed in a71841346. I wouldn't say it was a test. I always try to keep my diffs as small as possible. 😄